app/src/main/res/values/strings.xml

```diff
@@ -14,6 +14,9 @@
     <string name="settings_backup_location_none">None</string>
     <string name="settings_backup_location_internal">Internal storage</string>
     <string name="settings_backup_last_backup_never">Never</string>
+    <string name="settings_backup_dialog_title">Really turn off app backups?</string>
+    <string name="settings_backup_dialog_message">When you turn backups on again, the backup process may take longer than usual, and it will use additional storage space.</string>
```

Comment:
This is technically correct. I think it sounds ok too. We can change this in the future when we add restore set management, since then we'll have terminology for that and a way for the user to actually do something about it.